tests/cypress/integration/actions_objects2/case_10_polygon_shape_track_label_points.js

```diff
@@ -91,17 +91,48 @@ context('Actions on polygon', () => {
     });
 
     describe(`Testing case "${caseId}"`, () => {
-        it('Draw a polygon shape, track', () => {
+        it('Draw a polygon shape, track.', () => {
             cy.createPolygon(createPolygonShape);
             cy.createPolygon(createPolygonTrack);
         });
-        it('Draw a polygon shape, track with use parameter "number of points"', () => {
+
+        it('Draw a polygon shape, track with use parameter "number of points".', () => {
             cy.createPolygon(createPolygonShapePoints);
             cy.createPolygon(createPolygonTrackPoints);
         });
-        it('Draw a polygon shape, track with second label', () => {
+
+        it('Draw a polygon shape, track with second label.', () => {
             cy.createPolygon(createPolygonShapeSwitchLabel);
             cy.createPolygon(createPolygonTrackSwitchLabel);
         });
+
+        it('Set start point.', () => {
+            let notFirtsPointID;
+            cy.get('#cvat_canvas_shape_4')
+                .trigger('mousemove', {scrollBehavior: false})
```

Comment:
Global configuration for ``scrollBehavior`` in cypress.json is not an option?

https://docs.cypress.io/guides/references/configuration#cypress-json

Reply:
This value cannot be applied globally. Since there will be no scrolling in other cases. For example, when creating a task with Advanced configuration. In this case, Cypress will not scroll the screen down and the necessary fields will not be found.

Reply:
I would say that the most often behaviour should be set by default globally. If we need scrolling in 10 tests and disable it in 100 tests, it should be disabled by default. The decision is up to you.

Reply:
Yes, you're right. But I think it's better to do this as part of a separate task.